src/frontend/src/lib/components/dapps/DappPromoBanner.svelte

```diff
@@ -1,27 +1,23 @@
 <script lang="ts">
 	import Button from '$lib/components/ui/Button.svelte';
 	import Img from '$lib/components/ui/Img.svelte';
+	import ImgBanner from '$lib/components/ui/ImgBanner.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import type { FeaturedOisyDappDescription } from '$lib/types/dapp-description';
 	import { replacePlaceholders } from '$lib/utils/i18n.utils';
 
 	export let dAppDescription: FeaturedOisyDappDescription;
 </script>
 
-<article class="relative flex h-64 items-end overflow-hidden rounded-2xl">
+<article class="relative flex items-end overflow-hidden rounded-2xl">
 	{#if dAppDescription.screenshots.length > 0}
-		<div class="absolute inset-0 bg-brand-subtle-alt">
-			<Img
-				fitHeight={true}
-				height="100%"
-				width="100%"
-				styleClass="object-cover"
-				src={dAppDescription.screenshots[0]}
-				alt={replacePlaceholders($i18n.dapps.alt.website, { $dAppName: dAppDescription.name })}
-			/>
-		</div>
+		<ImgBanner
+			styleClass="bg-brand-subtle-alt max-h-64"
```

Comment:
ideally, we don't pass colors to components as general class. May you please put it outside `ImgBanner`?